libfwupd/fwupd-bios-setting.h

```diff
@@ -116,4 +155,17 @@ fwupd_bios_setting_get_id(FwupdBiosSetting *self);
 void
 fwupd_bios_setting_set_id(FwupdBiosSetting *self, const gchar *id);
 
+FwupdBiosAuthRole
+fwupd_bios_setting_get_auth_role(FwupdBiosSetting *self);
+void
+fwupd_bios_setting_set_auth_role(FwupdBiosSetting *self, FwupdBiosAuthRole role);
```

Comment:
the bit I'm struggling with is that these auth settings are not really per-attribute -- as I understand it you have one auth mechanism that allows you to "unlock" all the attributes -- right?